Documentation/5.4.1/README_U1_to_U3.rst

```diff
@@ -0,0 +1,1716 @@
+
+Run using
+
+::
+
+     cd /home/stack/nuage-tripleo-heat-templates/scripts/create_roles
+    ./create_compute_avrs_role.sh
+
+Above command will create a new ``ComputeAvrs``  role for your deployment, and compare it with sample `compute-avrs-role-sample.yaml <../../nuage-tripleo-heat-templates/roles/compute-avrs-role-sample.yaml>`_ .
+
+
+
+Phase 4: Generate a CMS ID for the OpenStack installation.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Cloud Management System (CMS) ID needs to be generated to configure your OpenStack installation with the VSD installation.
+
+Go to https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/generate-cms-id for the files and script to generate the CMS ID, and follow the instructions in README.md.
+
+The CMS ID is displayed in the output, and a copy of it is stored in a file called cms_id.txt in the same folder.
+
+Add the CMS ID to the neutron-nuage-config.yaml template file for the ``NeutronNuageCMSId`` parameter.
+
+
+Phase 5: Configure the basic Overcloud.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Follow the upstream OpenStack documentation *up to the step where* the ``openstack overcloud deploy`` command is run using the CLI or starting the Overcloud deployment (starting the Overcloud creation) in the UI.
+
+These are the OpenStack instructions:
+
+    * Configuring the container registry details: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/configuring-a-container-image-source
+    * Using the CLI: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/chap-configuring_basic_overcloud_requirements_with_the_cli_tools
+    * Using the UI: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/chap-configuring_basic_overcloud_requirements_with_the_ui_tools
+
+
+Phase 6: Check the Ironic node status to ensure that the Ironic nodes have been successfully created.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following commands.
+
+1. Run the following command. The results should show the *Provisioning State* status as *available* and the *Maintenance* status as *False*.
+
+::
+
+    openstack baremetal node list
+
+
+2. If profiles are being set for a specific placement in the deployment, run the following command. The results should show the *Provisioning State* status as *available* and the *Current Profile* status as *control* or *compute*.
+
+::
+
+    openstack overcloud profiles list
+
+
+Phase 7: Create the Heat templates.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to /usr/share/openstack-tripleo-heat-templates/environments/ on the Undercloud machine.
```

Comment:
change /usr/share/openstack-tripleo-heat-templates/environments/ to /home/stack/nuage-tripleo-heat-templates/environments/